src/codegen/sys/lib_.rs

```diff
@@ -294,18 +300,40 @@ fn generate_enums(w: &mut dyn Write, env: &Env, items: &[&Enumeration]) -> Resul
             let version = member_config.iter().filter_map(|m| m.version).next();
 
             if is_alias {
-                continue;
+                return quote! {};
             }
 
-            version_condition(w, env, version, false, 0)?;
-            writeln!(
-                w,
-                "pub const {}: {} = {};",
-                member.c_identifier, item.c_type, member.value,
-            )?;
+            let c_identifier = Ident::new(&member.c_identifier, Span::call_site());
+            let value = Literal::i32_unsuffixed(member.value.parse().unwrap());
+
+            let version_condition = match version {
+                Some(v) if v > env.config.min_cfg_version => {
+                    let v = v.to_cfg();
```

Comment:
Need different function, Ex. to_cfg_tokens(), as string not works in this case
```diff
-#[cfg(any(feature = "v3_18", feature = "dox"))]
+#[cfg(any("feature = \"v3_18\"", feature = "dox"))]
 pub const GDK_TOUCHPAD_SWIPE: GdkEventType = 41;
-#[cfg(any(feature = "v3_18", feature = "dox"))]
+#[cfg(any("feature = \"v3_18\"", feature = "dox"))]
 pub const GDK_TOUCHPAD_PINCH: GdkEventType = 42;
```

Reply:
Ah the string escaping :) Will take care of it once @GuillaumeGomez also agreed that this is the way forward.

In the end each of the functions would return a `TokenStream` and we would only write things out in the very end btw, instead of doing it in each function.